core/src/main/java/discord4j/core/object/entity/Webhook.java

```diff
@@ -656,13 +656,15 @@ public Mono<Message> editMessage(Snowflake messageId, WebhookMessageEditSpec spe
                     throw new IllegalArgumentException("Can't edit message with this webhook.");
                 }
 
+                boolean withComponents = !spec.components().isAbsent();
```

Comment:
based in docs i need make rest run with the `with_components` for allow use this, for the interaction this param its not necesary